pom.xml

```diff
@@ -419,7 +418,7 @@
       </dependency>
       <dependency>
         <groupId>${project.groupId}</groupId>
-        <artifactId>sonar-db</artifactId>
+        <artifactId>sonar-db-dao</artifactId>
```

Comment:
the dependency is the other way around

nevertheless, this is dependencyManagement, so both modules must be declared